### ISSUE TITLE
New (post schedule launch) scheduling requests

### ISSUE THREAD

**Issue by mherchel** [florida-drupalcamp/fldc-issues#87]
Related to #80

Hi guys!

I was just going to write when I saw the schedule email.

My availability for going to the camp is limited to only the Saturday. I'm flying out of town and already it is super expensive to book the flight 1 week before.

I would appreciate you could swap my session with another one with more flexible schedule on Saturday afternoon. I'll be landing at 9am on Orlando.

This is the session: https://www.fldrupal.camp/sessions/approved/florida-drupalcamp-2017/site-building/moving-our-company-site-drupal-7-8-sharing

Anytime in the afternoon of Saturday would be great.

Thanks!!!

**Comment by mherchel** [comment]
Hi Florida DrupalCamp Organizing team,

I am very sorry to say I have already committed to attending SANDCamp the same weekend in San Diego.  Apologies for not getting back to you sooner I am currently at LoopConf and I just saw your schedule posted and realized I had not replied yesterday. 

I really appreciate the acceptance and hate to back out, but plane tickets and hotels were already paid for :/  

Have an awesome camp and all the best,

Dwayne

On Mon, Feb 6, 2017 at 2:57 PM, <info@fldrupalcamp.org> wrote:
Hey mcdwayne,

Your session, "We Are All Making This Up: Improv Lessons For Developers", has
been approved!
